remme/tp/obligatory_payment.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2018 REMME
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+import logging
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_validator.journal.batch_injector import BatchInjector
+from sawtooth_validator.protobuf.batch_pb2 import (
+    Batch,
+    BatchHeader,
+)
+
+from sawtooth_sdk.protobuf.transaction_pb2 import (
+    Transaction,
+    TransactionHeader,
+)
+
+from remme.protos.obligatory_payment_pb2 import (
+    ObligatoryPaymentPayload,
+    ObligatoryPaymentMethod
+)
+
+from remme.protos.node_account_pb2 import (
+    NodeAccount
+)
+from remme.settings import (
+    CONSENSUS_ALLOWED_VALIDATORS,
+    SETTINGS_OBLIGATORY_PAYMENT,
+    SETTINGS_COMMITTEE_SIZE,
+)
+
+from remme.shared.forms import (
+    ObligatoryPaymentPayloadForm,
+)
+
+from .basic import (
+    PB_CLASS,
+    PROCESSOR,
+    VALIDATOR,
+    BasicHandler,
+    get_data,
+    get_multiple_data
+)
+from remme.shared.utils import hash512
+from remme.settings.helper import _get_setting_value
+LOGGER = logging.getLogger(__name__)
+
+FAMILY_NAME = 'obligatory_payment'
+FAMILY_VERSIONS = ['0.1']
+
+
+def withdraw_obligatory_payment(node_account, obligatory_payment):
+    if node_account.balance >= obligatory_payment:
+        node_account.balance -= obligatory_payment
+    elif node_account.reputation.unfrozen >= obligatory_payment:
```

Comment:
Put a blank line between elifs. Will be more readable.

```
if ...
    ....

elif ...
    ....

elif:
   ...
```

Reply:
Done.